File: TimBola.java
class TimBola{
    private String namaTim;
    private String negara;
    private String tahunBerdiri;
   
    TimBola(){    
    }


    class Pemain{
        private String pemain = null;

        Pemain(){
        }

        Pemain(String p ){
            this.pemain = p;
        }

        void setPemain(String p){
            this.pemain = p;
        }

        String getPemain(){
            return this.pemain;
        }

        

    }

    TimBola(String nT, String n, String tB){
        this.namaTim = nT;
        this.negara = n;
        this.tahunBerdiri = tB;
       
    }

    void setNama(String nT){
        this.namaTim = nT;
    }

    void setNegara(String n){
        this.negara = n;
    }

    void setTahunBerdiri(String tB){
        this.tahunBerdiri = tB;
    }

    

    String getNama(){
        return this.namaTim;
    }

    String getNegara(){
        return this.negara;
    }

    String getTahun(){
        return this.tahunBerdiri;
    }

   
    

}

File: TimBola.py
class TimBola :


    def __init__(self, nama = 0, negara = 0, tahun = 0):
        self.nama = nama
        self.negara = negara
        self.tahun = tahun

    class Pemain :
        def __init__(self, pemain):
            self.pemain = pemain

        def set_pemain(self, pemain):
            self.pemain = pemain

        def get_pemain(self):
            return self.pemain   
    
    def set_nama(self, nama):
        self.nama = nama    

    def set_negara(self, negara):
        self.negara = negara

    def set_tahun(self, tahun):
        self.tahun = tahun

    def get_nama(self):
        return self.nama

    def get_tahun(self):
        return self.tahun

    

    def get_negara(self):
        return self.negara

       
